core/cloud_connected_optimizer.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import time
+
+from core.utils.enums 					import ActionableEventTypes, RawEventTypes, MessageEventTypes
+from core.utils.enums 					import JobTypes, callback_handler_no_default
+from core.utils.dmljob 					import (DMLJob, DMLAverageJob, DMLCommunicateJob, DMLServerJob, \
+												DMLInitializeJob, DMLSplitJob, DMLTrainJob, DMLValidateJob)
+from core.utils.keras 					import serialize_weights, deserialize_weights
+from core.utils.dmlresult 				import DMLResult
+from core.blockchain.blockchain_utils	import TxEnum, content_to_ipfs
+from core.fed_avg_optimizer import FederatedAveragingOptimizer
+
+
+class CloudConnectedOptimizer(FederatedAveragingOptimizer):
+    def __init__(self, initialization_payload, dataset_manager):
+        super().__init__(initialization_payload, dataset_manager)
+        job_info = initialization_payload.get(TxEnum.CONTENT.name)
+        serialized_job = job_info.get('serialized_job')
+        self.job_data["uuid"] = serialized_job["uuid"]
```

Comment:
1. Aren't lines 17-18 unnecessary because of line 16? `self.job_data` should already be set...?
2. You should really s/uuid/job_uuid everywhere just to emphasize that it's job_uuid (because uuid by itself used to mean dataset_uuid)

Reply:
17-18 are just getting serialized_job from the initialization_payload so that line 19 can go off without a hitch. This optimizer requires a job_uuid. Will do for point 2).

Reply:
Ah I see, serialized_job isn't saved as a class attribute in the superclass

Reply:
2) Done